docs/docs-content/clusters/data-center/vmware.md

```diff
@@ -1072,6 +1049,18 @@ In addition to the default cloud account already associated with the private clo
 | **Username** | vCenter username|
 | **Password** | vCenter password|
 
+:::caution
+
+If you change the password for a user account in Vcenter, you must also change it in Palette for the same VMware cloud account. We suggest updating the passwords sequentially to avoid potentially locking Palette out of Vcenter.
+
+1. From the **Menu Menu** navigate to **Clusters** > **Add New Cluster** > **Deploy New Cluster** and select **VMware**. 
+2. Click on **Start VMware Configuration**.
+3. In the **Cloud Account** field, use the drop-down menu to choose the account with the changed password.
+4. Update the password in the **Password** field and click **Validate**. 
+5. Confirm your changes. 
```

Comment:
I recommend we move the lil mini-how-to to it's own section. This can be a `###` heading or a `## ` you call.
That way, we can link to this section from the callout box.
And we can link to this section in a future troubleshooting document. (Out of scope for now).
The validation part can be a sentence that explains the validation is done by the Validation button and incorrect credentials will result in an error. We can also advice the user to attempt to deploy a cluster as well.
Prereqs would be to have access to the vCenter credentials

Reply:
Incorporated comments.